index.md

```diff
@@ -1,69 +1,36 @@
 ---
-title: "Documentation for The Canonical Distribution of Kubernetes <sup>&reg;</sup>"
```

Comment:
This page looks a little sparse now it's in a single column. I guess there is an argument for rewriting it if it is to be part of u.c anyhow